index.html

```diff
@@ -2,17 +2,212 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Hello world</title>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/style.css">
+  <script src="https://kit.fontawesome.com/1ada02b4f0.js" crossorigin="anonymous"></script>
+  <title>LEEHAR'S FASHION WEEK </title>
 </head>
 
 <body>
-    <header>
-        <h1 class="header">Hello world!</h1>
-    </header>
+  <header>
+    <div class="top-navbar">
+      <ul>
+        <li>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa fa-facebook"></i></a>
+        </li>
+        <li><a href="#">Celebrities</a></li>
+        <li><a href="#">Vlog</a></li>
+        <li><a href="#"> Logout</a></li>
+      </ul>
+    </div>
+    <div class="main-navbar flex-center">
+      <div class="logo">
+        <a href="#"><img src="./assets/creative-tailor.png" alt="fashion logo" /></a>
+      </div>
+      <nav class="navbar">
+        <div class="hamburger">
+          <img src="./assets/hamburger.svg" alt="Menu Hamburger" />
+        </div>
+        <ul class="nav-links">
+          <li class="close-menu-btn">
+            <button type="submit">
+              <span>&times;</span>
+            </button>
+          </li>
+          <li><a href="index.html" id="current">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Models</a></li>
+          <li><a href="#">Tickets</a></li>
+          <li id="leehar-line-lg" class="flex-center"><a href="#">LEEHAR'S LINE</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <section class="home-section">
+    <div class="home-wrapper">
+      <div class="home-section-header">
+        <h2 class="home-message">"Hello! Fashionista"</h2>
+        <h1 class="home-info">
+          It's Leehar's fashion week🎉👙👗 where: Fashion is like eating, you shouldn't stick to the same menu😉👌
+        </h1>
+      </div>
+      <div class="home-text flex-center">
+        <p>
+          Create your own visual style... let it be unique for yourself and yet identifiable for others because Style is
+          a reflection of your attitude and your personality.
+        </p>
+      </div>
+      <div class="opening-hours">
+        <p>2021.12.17(FRI) ~ 19(SUN)</p>
+      </div>
+      <div class="home-text-2">
+        <p>
+          @ EKO Hotel and suites, Oriental Hotel and more.
+        </p>
+      </div>
```

Comment:
- [ ] Since your background is dark, it would be better to use `white` color in the text for clear visibility

Reply:
Thanks for pointing this out, I totally missed it.